lms/tests.py

```diff
@@ -1,17 +1,20 @@
 """Tests for the lms module itself."""
 
+import logging
 import mimetypes
 
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 from mock import patch
 
 from edxmako import LOOKUP, add_lookup
-from lms import startup
+from microsite_configuration import microsite
```

Comment:
Was this the only place `lms.startup.enable_microsites` was referenced?  If so, the function is orphaned.

Reply:
Yes, indeed. I will remove it.